### DISCUSSION TITLE
Outline should add more extended Markdown | Footnotes, Definition Lists, Superscript, and Subscript

### DISCUSSION THREAD

**Discussion by snarkyalyx** [outline/outline#5016] [Ideas]
Hello everyone!

I always wondered why Outline has some extended Markdown functionality, but lacks some others. Most notably would be the lack of Footnotes.

Currently, Outline lacks:
**Footnotes**: Here's a simple footnote example.[^1] Unfortunately, Outline does not support this in the slightest.
**Definition Lists**: You can define a term, and then have individual, multiple definitions. This is like a bulleted list without a dot.
**Superscript and Subscript**: This lets you write things raised or lowered. For example, x**²** being superscript, and H**₂**O being subscript.
**Automatic Linking**(ish): Typing in a URL manually doesn't automatically link it, only when it was copied in. (may be a bug?)

Considering that LaTeX was recently added, you may note that this is going to bring further growth for STEM-related users and businesses.

[^1]: This is the footnote example.

**Comment by wouter-deen** [comment]
Would really love to see subscript and superscript support. That should not be that hard to implement.

**Comment by snarkyalyx** [comment]
Also, bump - Any updates on this? Is this considered?

**Reply by tommoor** [reply]
Happy to entertain any of these ideas, I think they have to built into the editor properly though – not just through markdown
